configure.ac

```diff
@@ -175,15 +175,17 @@ if test "x$enable_daemon" = "xyes"; then
   AC_SUBST(LIBATASMART_CFLAGS)
   AC_SUBST(LIBATASMART_LIBS)
 
-  PKG_CHECK_MODULES(LIBMOUNT, [mount >= 2.30],
+  PKG_CHECK_MODULES(LIBMOUNT, [mount >= 2.18])
```

Comment:
We should consider always requiring 2.30, it was released nearly 3 years ago so it should be available in all new distributions. Even Debian stable has 2.33. (But not RHEL/CentOS 7, so 2.9.0 wouldn't work there.)

Reply:
Yeah, that was my point, keep support for RHEL7 in the meantime. There's no real reason to be that strict, at least for one more cycle. We may reconsider that for UDisks 2.10 perhaps with libblockdev-3.0 coming but for the moment I'd suggest to keep it this way.